evalutils/cli.py

```diff
@@ -39,6 +40,25 @@ def validate_python_module_name_string(ctx, param, arg):
     return validate_python_module_name_string
 
 
+class AbbreviatedChoice(click.Choice):
+    def __init__(self, choices: List[str]):
+        super().__init__(choices=choices, case_sensitive=True)
+        self.abbreviations = [e[0] for e in choices]
```

Comment:
The abbreviations must be unique, and based on L56 abbreviations must be a list so you cannot use a set. Can you add validation and a test to this?

Reply:
Yes, I'll do that.

Reply:
Done